src/app/application/components/route-view/RouteView.vue

```diff
@@ -71,6 +86,31 @@ if (!hasParent) {
   provide('route-view-parent', routeView)
 }
 
+const iParent: ImmediateParent | undefined = inject('route-view-immediate-parent', undefined)
```

Comment:
**Suggestion**: This avoids the two TypeScript errors ignored in this file.

```suggestion
const iParent = inject<ImmediateParent | undefined>('route-view-immediate-parent', undefined)
```

Reply:
>> Typescript tells me this can never happen, javascript tells me otherwise.

>>  Maybe I'm doing something wrong, but I didn't want to spend to much time looking into it right now, seeing as I'm coming back here after the PR is merged, but if its a simple "oh you just need to...." lmk

> Suggestion: This avoids the two TypeScript errors ignored in this file.

Ah of course! ty! Weird it only needs it here an not the one above, which I'm assuming would be better using the same form 🤔 . Lemme change this one up before merging.